packages/remix/README.md

```diff
@@ -71,28 +71,31 @@ import {
   ScrollRestoration,
 } from "@remix-run/react";
 
-import { ErrorBoundary, withSentryRouteTracing } from "@sentry/remix";
+import { withSentryRouteTracing } from "@sentry/remix";
 
 function App() {
   return (
-    <ErrorBoundary>
-      <html>
-        <head>
-          <Meta />
-          <Links />
-        </head>
-        <body>
-          <Outlet />
-          <ScrollRestoration />
-          <Scripts />
-          <LiveReload />
-        </body>
-      </html>
-    </ErrorBoundary>
+    <html>
+      <head>
+        <Meta />
+        <Links />
+      </head>
+      <body>
+        <Outlet />
+        <ScrollRestoration />
+        <Scripts />
+        <LiveReload />
+      </body>
+    </html>
   );
 }
 
-export default withSentryRouteTracing(App);
+export default withSentryRouteTracing(App, {
+  wrapWithErrorBoundary: false // default: true
```

Comment:
```suggestion
```

We shouldn't have the docs saying that it's disabled. We can just leave a note that they can use the `wrapWithErrorBoundary` to disable if needed.